jenkinsfiles/build.json

```diff
@@ -5,7 +5,7 @@
             "storePassword": "android",
             "alias": "android",
```

Comment:
Alias should be `dummyalias`. 
It doesn't work with 'android', and in the project linked from description it's also 'dummyalias'.